test.sh.in

```diff
@@ -11,15 +11,21 @@ if [ ! -z "${5}" ]; then
 fi
 MODIFY_API_PACKAGE="${MODIFY_API_PACKAGE}"
 
-cleanupVMs(){
-    vagrantDestroy
-    # Suspend any other running vagrant boxes
-    vagrantSuspendAll
+cleanUpDockerImages(){
+    echo $SUDO_PASSWORD |sudo -S docker rmi $(echo $SUDO_PASSWORD |sudo -S docker images -q)
```

Comment:
Please just rmi docker images that this test generated.
Other images may be useful for other test